incubator/hnc/pkg/forest/forest.go

```diff
@@ -46,12 +48,16 @@ func (f *Forest) Get(nm string) *Namespace {
 	if ok {
 		return ns
 	}
-	ns = &Namespace{forest: f, name: nm, children: namedNamespaces{}}
+	ns = &Namespace{forest: f, name: nm, children: namedNamespaces{}, conditions: conditions{}}
```

Comment:
Maybe this is getting long enough to split into multiple lines?

```
ns = &Namespace{
  forest: f,
  name: nm,
  children: namedNamespaces{},
  conditions: conditions{},
}
```

Reply:
done